File: java/cuoiki/src/mypack/PHAT.java
package mypack;

public class PHAT {
	public SACH MSSach;
	public DOCGIA MSDG;
	public String LyDoPhat;
	public double TienNop;
	public PHAT(SACH mSSach, DOCGIA mSDG, String lyDoPhat, double tienNop) {
		super();
		MSSach = mSSach;
		MSDG = mSDG;
		LyDoPhat = lyDoPhat;
		TienNop = tienNop;
	}
	public SACH getMSSach() {
		return MSSach;
	}
	public void setMSSach(SACH mSSach) {
		MSSach = mSSach;
	}
	public DOCGIA getMSDG() {
		return MSDG;
	}
	public void setMSDG(DOCGIA mSDG) {
		MSDG = mSDG;
	}
	public String getLyDoPhat() {
		return LyDoPhat;
	}
	public void setLyDoPhat(String lyDoPhat) {
		LyDoPhat = lyDoPhat;
	}
	public double getTienNop() {
		return TienNop;
	}
	public void setTienNop(double tienNop) {
		TienNop = tienNop;
	}
	
}

File: DoAnOOP/Phat.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DoAnOOP
{
    public class Phat
    {
        public Sach MSSach { get; set; }
        public DocGia MSDG { get; set; }
        public string LyDoPhat { get; set; }
        public double TienNop { get; set; }

        public Phat(Sach MSSach, DocGia MSDG, string LyDoPhat, double TienNop)
        {
            this.MSSach = MSSach;
            this.MSDG = MSDG;
            this.LyDoPhat = LyDoPhat;
            this.TienNop = TienNop;
        }
        public Phat(Phat phat)
        {
            MSSach = phat.MSSach;
            MSDG = phat.MSDG;
            LyDoPhat = phat.LyDoPhat;
            TienNop = phat.TienNop;
        }
    }
}
